source/wisun_br_conn_handler.c

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Could wisun_br_conn_handler.c functionality be located in borderrouter_ws.c and file removed?